addons/l10n_ec/data/l10n_latam_document_type_data.xml

```diff
@@ -0,0 +1,404 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	<data noupdate="0"> <!-- #TODO poner no update = 1 al completar la version 13 -->
+    	<record id="ec_02" model="l10n_latam.document.type">
+			<field name="code">02</field>
+			<field name="name">Nota o Boleta de Venta</field>
+			<field name="report_name">Nota de Venta</field>
+			<field name="doc_code_prefix">Not</field>
+			<field name="sequence">25</field>
+			<field name="internal_type">invoice</field>
+			<field name="l10n_ec_type">in_invoice</field>
+			<field name="l10n_ec_authorization">third</field>
+			<field name="country_id" ref="base.ec"/>
+		</record>
+
+		<record id="ec_03" model="l10n_latam.document.type">
+			<field name="code">07</field>
+			<field name="name">Retención de Venta</field>
+			<field name="report_name">Comprobante de Retención</field>
+			<field name="doc_code_prefix">Ret</field>
+			<field name="sequence">65</field>
+			<field name="internal_type"></field>
+			<field name="l10n_ec_type">out_withhold</field>
```

Comment:
do we need internal_type and l10n_ec_type?  We can probably do selection_add on the internal_type